app/views/confirm.scala.html

```diff
@@ -13,6 +13,9 @@
   }
   <h1 class="form-title heading-xlarge" id="pageHeading">@{journeyData.resolvedConfig.confirmPage.heading}</h1>
   @if(selectedAddress.isDefined) {
+    @if(journeyData.resolvedConfig.confirmPage.showConfirmChangeText){
```

Comment:
Shouldn't this go just before the submit button? Rather than at the top of the view?

Reply:
I made the changes on a different PR since I didn't have access to mitchells
https://github.com/hmrc/address-lookup-frontend/pull/31